examples/suit_update/main.c

```diff
@@ -54,24 +74,22 @@ int main(void)
         printf("[FAILED] You're not running riotboot\n");
     }
 
-    /* nanocoap_server uses gnrc sock which uses gnrc which needs a msg queue */
-    msg_init_queue(_main_msg_queue, MAIN_QUEUE_SIZE);
-
-    puts("Waiting for address autoconfiguration...");
-    xtimer_sleep(3);
```

Comment:
There is an issue with removing this, without giving time for address to configure the first time `ifconfig` is called the address is tentative `TNT[1]`, this causes the wrong address to be used (not the EUI64) address which the node uses to respond to COAP requests.

So keep `xtimer_sleep` or change the matching regex (since ping will make sure the node becomes available).

```
> ifconfig
Iface  5  HWaddr: 00:53:CB:6B:A0:79 
          L2-PDU:1500 MTU:1500  HL:64  RTR  
          RTR_ADV  
          Source address length: 6
          Link type: wired
          inet6 addr: fe80::253:cbff:fe6b:a079  scope: link  TNT[1]
          inet6 addr: fe80::2  scope: link  VAL
pinging node...
PING fe80::2%riot0(fe80::2%riot0) 56 data bytes
```

Reply:
or add a sleep(1) in the test script.

Reply:
Fixed